components/StartEndCollection.vue

```diff
@@ -21,53 +21,39 @@
 <script>
 // TODO Validation - end date > start date, no stupidly long events (3 days?), no overlapping dates, only one blank slot.
 // TODO This code could do with a bit of extra testing - various combinations of add/edit/cancel/delete etc.
-const StartEndDate = () => import('~/components/StartEndDate')
+import StartEndDate from '~/components/StartEndDate'
```

Comment:
We don't need dynamic imports for _everything_ especially in cases like this, where `StartEndDate` is only used in this component.